auto_submit/lib/requests/github_webhook.dart

```diff
@@ -39,9 +43,9 @@ class GithubWebhook extends RequestHandler {
     print('pullRequest: ${pullRequest.id}');
 
     if (hasAutosubmit) {
-      log.info(body);
+      log.info('Find one that has autosubmit label $body');
       // TODO(kristinbi): Publish the pr with 'autosbumit' label to pubsub.
-
+      await pubsub.publish('auto-submit', pullRequest);
```

Comment:
Let's call the topic `auto-submit-topic`. Can you create an issue and assign it to me for creating the PubSub topic and subscription in GCP

Reply:
Sure Casey, I created a new topic with the suggestted name.